docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -1180,6 +1180,13 @@ <h1>Samza Configuration Reference</h1>
                     </td>
                 </tr>
 
+                    <td class="property" id="streams-samza-broadcast">streams.<span class="stream">stream-id</span>.<br>samza.broadcast</td>
```

Comment:
I think this is the old config page. Should add it to the md file instead.

Reply:
Oh, ok